### PR TITLE
Add version info to kube-scheduler, kube-proxy and kubelet logs.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:
#46047 add such info to apiserver and CM. This adds version info to other binaries.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: related to #46047 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by gyliu513** [comment]
@k8s-bot ok to test

**Comment by xiangpengzhao** [comment]
ping for reviewing and approval. Thanks!

**Review by dcbw — APPROVED** [review]
Seems useful.

**Comment by xiangpengzhao** [comment]
Thanks @dcbw !
/assign @davidopp @mikedanese
for approval

**Comment by xiangpengzhao** [comment]
/unassign @mikedanese
/assign @lavalamp
for `cmd/` approval. Thanks!

**Comment by jayunit100** [comment]
Do we want to add flag info here also ? I think in the other PR they did that ? https://github.com/kubernetes/kubernetes/pull/46047/files#diff-3aace893bfe3becf7f78d8c56b281344R51.

otherwise looks good.

**Comment by xiangpengzhao** [comment]
@jayunit100 thanks for pointing this out! I check the code again and find that `InitFlags` you mentioned has already been invoked in the `main` func of each components.

e.g.:
https://github.com/xiangpengzhao/kubernetes/blob/fb7eebc85f1a65be1e458b439601603e137b5640/cmd/kubelet/kubelet.go#L42
```go
func main() {
	s := options.NewKubeletServer()
	s.AddFlags(pflag.CommandLine)

	flag.InitFlags()
	logs.InitLogs()
        ... ...
}
```

BTW, `kube-proxy` is an exception. There is a TODO:
https://github.com/xiangpengzhao/kubernetes/blob/fb7eebc85f1a65be1e458b439601603e137b5640/cmd/kube-proxy/proxy.go#L35-L40
```go
	// TODO: once we switch everything over to Cobra commands, we can go back to calling
	// utilflag.InitFlags() (by removing its pflag.Parse() call). For now, we have to set the
	// normalize func and add the go flag set by hand.
	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
	// utilflag.InitFlags()
```

**Comment by xiangpengzhao** [comment]
Rebased. Folks PTAL. Thanks!

File: cmd/kube-proxy/app/server.go [hunk 1]
```diff
@@ -421,6 +422,8 @@ func createClients(config componentconfig.ClientConnectionConfiguration, masterO
 
 // Run runs the specified ProxyServer.  This should never exit (unless CleanupAndExit is set).
 func (s *ProxyServer) Run() error {
+	// To help debugging, immediately log version
```
Comment (mikedanese) [inline cmd/kube-proxy/app/server.go]: this comment is not helpful.

Comment (xiangpengzhao) [reply cmd/kube-proxy/app/server.go]: yeah, but I'd like to leave it as-is to keep consistent with what #46047 did :)

**Pull request merged**